### PR TITLE
Support Giphy mp4s. Play non-html5 gifs inline

### PR DESCRIPTION
**Pull request by schwers**

### CODE REVIEW

File: src/views/components/ListingContent.jsx [hunk 1]
```diff
@@ -27,6 +28,12 @@ function _gifToHTML5(url) {
     };
   }
 
+  if (url.indexOf('giphy.com') > -1) {
```
Comment (ajacksified) [inline src/views/components/ListingContent.jsx]: We should start replacing `indexOf(domain)` with checks using `url.parse` for safety.

**Comment by ajacksified** [comment]
One note (from slack conversation) - :nail_care:

**Comment by schwers** [comment]
Fixes gifToHTML5 to be more secure and adds tests.
Adds support for giphy gifs. 
Plays all other gifs we don't know about inline. 
Higher res gfycat previews

image

is now

image

**Comment by ajacksified** [comment]
:fish: assuming green

**Pull request merged**